apps/client/src/shared/apis/timerApi.ts

```diff
@@ -13,3 +18,33 @@ export const getActiveReminderTimer =
     );
     return response.data;
   };
+
+export const getDailyReminderCards = async (
+  limit = 12,
+  offset = 0
+): Promise<DailyReminderResponse> => {
+  const response = await apiRequest.get<DailyReminderResponse>(
+    `/api/daily-reminders?limit=${limit}&offset=${offset}`
+  );
+  return response.data;
+};
```

Comment:
response에서 data만 쓴다면 return 받을 때부터 구조 분해 할당으로 받아올 수 있어요!

```suggestion
  const { data } = await apiRequest.get<DailyReminderResponse>(
    `/api/daily-reminders?limit=${limit}&offset=${offset}`
  );
  return data;
```

Reply:
넵! 아직은 API 완성본이 아니라 목데이터로 테스트 중인 상태예요 :)
실제 API 나오면 그에 맞춰 로직 정리해볼 예정입니다! 실제 연동되면 다시 정리할게요!

Reply:
넵 좋습니다 👍